lib/faraday/options/env.rb

```diff
@@ -126,18 +123,18 @@ def body=(value)
 
     # @return [Boolean] true if status is in the set of {SuccessfulStatuses}.
     def success?
-      SuccessfulStatuses.include?(status)
+      Env::SuccessfulStatuses.include?(status)
```

Comment:
I'm a bit surprised these `Env::` are necessary, but it's not the end of the world 😄